.github/workflows/nestbuildmatrix.yml

```diff
@@ -47,6 +50,9 @@ jobs:
            - xNEST_BUILD_TYPE: "OPENMP_ONLY"
              cpp_compiler: "gcc"
              os: macos-latest
+           - xNEST_BUILD_TYPE: "ARBOR_BACKEND"
+             cpp_compiler: "gcc"
+             os: macos-latest
```

Comment:
Insert below here
```
   - xNEST_BUILD_TYPE: "ARBOR_BACKEND"
              cpp_compiler: "clang"
              os: macos-latest
```
to completely prevent Arbor testing on macOS. At present, we only perform a minimal test on macOS to limit resource use; not all necessary requirements for a full test are installed.